docs/en/database_engines/lazy.md

```diff
@@ -0,0 +1,11 @@
+# Lazy
+
+Works like `Ordinary`, but keeps tables in RAM only `expiration_time_in_seconds` seconds after last access. Can be used only with \*Log tables.
+
+It's optimized for storing many small \*Log tables, for which there is a long time interval between accesses.
+
+## Creating a Database
+
+```
+CREATE DATABASE testlazy ENGINE = Lazy(expiration_time_in_seconds);
```

Comment:
Maybe `ENGINE = Lazy(INTERVAL 3 SECONDS)` is better ?

Reply:
If we do this way, we also have to change it for Buffer table.